src/pubsub/ua_pubsub.c

```diff
@@ -4,8 +4,10 @@
  *
  * Copyright (c) 2017-2018 Fraunhofer IOSB (Author: Andreas Ebner)
  * Copyright (c) 2019 Fraunhofer IOSB (Author: Julius Pfrommer)
+ * Copyright (c) 2019 Kalycito Infotech Private Limited
  */
 
+#include <open62541/plugin/log_stdout.h>
```

Comment:
Don't add arch-specific plugins in the main code.
Use the logger from the server config instead.